src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -18786,6 +18786,57 @@ def test_aks_disable_ssh(self, resource_group, resource_group_location):
         # delete
         self.cmd('aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='centraluseuap')
+    def test_aks_entraid_ssh(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=141402123&view=results) to validate the change.